tensorboard/plugins/profile/pod_viewer/pod_viewer_dashboard/pod-viewer-dashboard.ts

```diff
@@ -0,0 +1,232 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+namespace pod_viewer_dashboard {
+
+Polymer({
+  is: 'pod-viewer-dashboard',
+  properties: {
+    _data: {
+      type: Object,
+      notify: true,
+    },
+    podStats: {
```

Comment:
The component is used in tf-profile-dashboard and it sets only handful of properties (`_data` and `active`). Can you please turn these properties to private ones and use computed properties for the ones that derive information from `_data`?

Reply:
Done.